server/helpers/ffmpeg-utils.ts

```diff
@@ -199,9 +192,9 @@ function getVideoFileStream (path: string) {
  * and quality. Superfast and ultrafast will give you better
  * performance, but then quality is noticeably worse.
  */
-function veryfast (_ffmpeg) {
-  _ffmpeg
-    .preset(standard)
+async function presetH264VeryFast (ffmpeg, resolution: VideoResolution, fps: number): ffmpeg {
```

Comment:
Please type `ffmpeg`

Reply:
What do you mean? Not sure how this works in Typescript..

Reply:
Oh just noticed it, I thought you mean something about the return type.

Reply:
:+1: